managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3569,6 +3570,10 @@ private ManagedLedgerInfo getManagedLedgerInfo() {
         return buildManagedLedgerInfo(ledgers);
     }
 
+    private ManagedLedgerInfo getManagedLedgerInfo(NavigableMap<Long, LedgerInfo> ledgersTmp) {
+        return buildManagedLedgerInfo(ledgersTmp);
```

Comment:
The `ledgersTmp` can be avoided by extending `buildManagedLedgerInfo`. Just add the new LedgerInfo after line 3578.

Reply:
Why do we need to create a new method?

Reply:
OK

Reply:
Fixed.PTAL,thanks! @Jason918